lib/java/ydoc-server/src/test/java/org/enso/ydoc/server/YdocTest.java

```diff
@@ -18,15 +18,15 @@
 import java.util.concurrent.Executors;
 import java.util.concurrent.LinkedBlockingQueue;
 import java.util.concurrent.TimeUnit;
-import org.enso.ydoc.jsonrpc.JsonRpcRequest;
-import org.enso.ydoc.jsonrpc.JsonRpcResponse;
-import org.enso.ydoc.jsonrpc.model.ContentRoot;
-import org.enso.ydoc.jsonrpc.model.FilePath;
-import org.enso.ydoc.jsonrpc.model.FileSystemObject;
-import org.enso.ydoc.jsonrpc.model.WriteCapability;
-import org.enso.ydoc.jsonrpc.model.result.FileListResult;
-import org.enso.ydoc.jsonrpc.model.result.InitProtocolConnectionResult;
-import org.enso.ydoc.jsonrpc.model.result.TextOpenFileResult;
+import org.enso.ydoc.server.jsonrpc.JsonRpcRequest;
```

Comment:
Wow: So many `jsonrpc` classes just for testing purposes!

I wasn't aware we have so many classes related to _mock communication with language server_. Do they _mock the communication_? What do they actually test?

Reply:
They need to emulate initialization message exchange with the language server